test/integration/event_page_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'test_helper'
+
+feature 'Event' do
+  def setup
+    @user = make_user
+    @admin = make_admin
+  end
+
+  test 'shows a logo image if the event has a logo' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true, logo: 'logo-image')
+
+    visit events_path
+
+    assert page.text.include?('The Event')
+    assert page.body.include?('logo-image')
+    assert_not page.body.include?('event-default')
```

Comment:
I would like to include a regex here that matches event-default-[string].png as it is in fact what we are exactly checking.

Reply:
assert_match page.body, /logo-image-.*\.png/
assert_match page.body, /logo-image-\w*\.png/